### PR TITLE
Soft remove SIWE

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref DEV-2300

### CODE REVIEW

File: pkg/lib/config/environment.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ type WebAppCDNHost string
 
 type CORSAllowedOrigins string
 
-type NFTIndexerAPIEndpoint string
+type Deprecated_NFTIndexerAPIEndpoint string
```
Comment (tung2744) [inline pkg/lib/config/environment.go:21]: Why do we need to keep this type?

File: pkg/lib/config/config.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ type AppConfig struct {
 	Verification              *VerificationConfig   `json:"verification,omitempty"`
 	Deprecated_OTP            *OTPLegacyConfig      `json:"otp,omitempty"`
 
-	Web3 *Web3Config `json:"web3,omitempty"`
+	Web3 *Deprecated_Web3Config `json:"web3,omitempty"`
```
Comment (tung2744) [inline pkg/lib/config/config.go:87]: Also rename `Web3` to `Deprecated_Web3`?

File: pkg/lib/config/feature.go [hunk 3]
```diff
@@ -44,7 +44,7 @@ type FeatureConfig struct {
 	RateLimits       *RateLimitsFeatureConfig       `json:"rate_limits,omitempty"`
 	Messaging        *MessagingFeatureConfig        `json:"messaging,omitempty"`
 	Collaborator     *CollaboratorFeatureConfig     `json:"collaborator,omitempty"`
-	Web3             *Web3FeatureConfig             `json:"web3,omitempty"`
+	Web3             *Deprecated_Web3FeatureConfig  `json:"web3,omitempty"`
```
Comment (tung2744) [inline pkg/lib/config/feature.go:47]: Also rename `Web3` to `Deprecated_Web3`?

File: pkg/lib/config/environment.go (hunk 1 above)
Comment (louischan-oursky) [reply pkg/lib/config/environment.go:21]: You're right. Will remove it.

**Comment by louischan-oursky** [comment]
Added 3 commits to address the comments.

**Review by tung2744 — APPROVED** [review]

**Pull request merged**